lib/elixir/pages/Guards.md

```diff
@@ -173,15 +179,18 @@ defmodule Check do
   def empty?(_val), do: false
 end
 
-Check.empty?(%{}) #=> true
-Check.empty?({}) #=> true
+Check.empty?(%{})
+#=> true
+
+Check.empty?({})
+#=> true
 ```
 
 ## Defining custom guard expressions
 
-As mentioned before, only the expressions listed in this page are allowed in guards. However, we can take advantage of macros to write custom guards that can simplify our programs or make them more domain-specific. At the end of the day, what matters is that the *output* of the macros (which is what will be compiled) boils down to a combinations of the allowed expressions.
+As mentioned before, only the functions and operators listed in this page are allowed in guards. However, we can take advantage of macros to write custom guards that can simplify our programs or make them more domain-specific. At the end of the day, what matters is that the *output* of the macros is boiled down to a combination of allowed expressions.
```

Comment:
"expressions" is better than functions and operators, as some of them may be macros.

Also, please avoid passive voice.

Reply:
should we then correct the name of the title of the first section to "List of allowed functions, macros and operators" ?

Reply:
done